packages/flutter/lib/src/widgets/_platform_selectable_region_context_menu_web.dart

```diff
@@ -2,11 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:html' as html;
 import 'dart:ui' as ui;
 
 import 'package:flutter/rendering.dart';
+import 'package:js/js.dart';
 
+import '../dom.dart';
```

Comment:
Maybe cc @chunhtai so he's aware of the changes that are happening to dom:html vs dom.dart? It seems he may touch these files again :)

Reply:
Good call!